### ISSUE TITLE
Depth of Root Node is 0 not 1

### ISSUE THREAD

**Issue by trialsNow** [LeetCode-Feedback/LeetCode-Feedback#5392]
Check this out
https://leetcode.com/explore/learn/card/data-structure-tree/17/solve-problems-recursively/534/

Depth of Root Node is 0 not 1

Source : https://en.wikipedia.org/wiki/Tree_(data_structure)

**Comment by LC-Sue** [comment]
Hi @trialsNow 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Sue** [comment]
Hi @trialsNow,
Thank you for sharing your concerns. I agree with you, by convention it makes more sense for the root node to be given a depth of zero. However, for the purpose of this material - to better understand the basics of recursion - whether we define the depth as 0 or 1 makes little difference. Given that the entire animation would need to be updated to address this inquiry, we do not plan to implement this change throughout the page. However, we have added a note when defining the depth of the root node to inform users that this is just how we defined the depth here, but by convention, the depth is often defined as 0.
